.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
+install: bundle install --retry=3 --deployment --without development production
 cache: bundler
-sudo: true
+sudo: false
```

Comment:
Tonmoy said we don't need `sudo` anymore because we are no longer installing ClamAV via `apt-get` or trying to run `freshclam`.